examples/openssl/src/main.rs

```diff
@@ -41,15 +41,15 @@ pub fn main() -> Result<(), Error> {
 
     let listener = TcpListener::bind(&addr)?;
 
+    let mut runtime = Runtime::new()?;
```

Comment:
I'm not worried about it, but I'm curious why this move happened?

Reply:
I think it's unnecessary.  During the refactor, I originally passed the runtime to `bind_server`, but later refactored it out when I made `bind_server` an async function instead.